plugin/storage/es/dependencystore/storage.go

```diff
@@ -31,46 +32,61 @@ import (
 )
 
 const (
-	dependencyType  = "dependencies"
-	dependencyIndex = "jaeger-dependencies-"
+	dependencyType       = "dependencies"
+	dependencyIndex      = "jaeger-dependencies-"
+	indexPrefixSeparator = "-"
 )
 
 // DependencyStore handles all queries and insertions to ElasticSearch dependencies
 type DependencyStore struct {
-	ctx         context.Context
-	client      es.Client
-	logger      *zap.Logger
-	indexPrefix string
+	ctx                   context.Context
+	client                es.Client
+	logger                *zap.Logger
+	dependencyIndexPrefix string
+	useReadWriteAliases   bool
+}
+
+// DependencyStoreParams holds constructor parameters for NewDependencyStore
+type DependencyStoreParams struct {
+	Client              es.Client
+	Logger              *zap.Logger
+	MetricsFactory      metrics.Factory
+	IndexPrefix         string
+	UseReadWriteAliases bool
```

Comment:
Could you please remove this and keep only mappings templates handling in this PR?

Reply:
I pushed a fresh commit only containing the removal of the indices